index.md

```diff
@@ -244,40 +244,46 @@ FIXME
+    If data and files are stored in a standard way,
+    your future self can come back to a project after a long field season and not have to spend days remembering what you did last, or what your collaborators have changed while you have been gone.
+    This is also true for someone else who wants to take what you've done and go in a new direction.
 *   *Fixability:*
-    version control of code, figures, and data helps you figure out why Figure 4 looks different now from last week.
+    Having access to every version of code, figures, and data helps you figure out why Figure 4 looks different now from last week.
 *   *Sharing and Collaboration:*
-    version control tools make it easy to share projects and update them, sometimes simultaneously, with other collaborators
+    Version control tools make it easy to share projects and update them, sometimes simultaneously, with other collaborators.
 
 ### Rules
 
-1.  Everything created by a human being goes under version control as soon as it's created
-    *   With the possible exception of manuscripts (discussed below)
-    *   And the possible exception of raw data, especially if large
-2.  The repository is mirrored on at least one machine that *isn't* the researcher's computer
-    *   E.g., pushed to GitHub or sync'd with a departmental server
-3.  The project repository contains a checklist of things that must pass before a change is shared with the world
-    *   Style guidelines met, to-do list updated, smoke test(s) pass
-    *   Note: "shared with the world" means "pushed to GitHub" or however else changes are copied off the researcher's computer
+1.  Everything created by a human being goes under version control as soon as it is created. This includes scripts for data management, cleaning and analyses; software packages; and documentation. Some types of files may not be appropriate for version control:
+  * raw and / or synthesized data (see section on data, below)
+  * manuscripts, depending on the format, but see the Manuscript section for details  
+  * intermediate files that can be regenerated
+2.  The repository is mirrored on at least one machine that *isn't* the researcher's computer. The other machine might be a lab or departmental server, or a public site such as GitHub or Bitbucket.
+3. You share your changes frequently, and incorporate changes from others frequently. Don't allow your local version of the project repository to drift apart from copies on other machines.  
+3.  Decide (along with any collaborators) on your own checklist for committing & sharing changes to the project and document this checklist in the repository. ('Sharing' means moving changes from your computer onto another machine / site). The list might include things such as:
+    * Writing commit messages that clearly explain any changes
+    * Size and content of single commits
+    * Style guidelines for code
+    * Updating to-do lists
+    * Bans on committing half-done work or broken code
```

Comment:
Before we ship, we should include pointers to a couple of checklists people can use as starting points.